torch/_inductor/compile_fx.py

```diff
@@ -464,6 +464,12 @@ def compile_fx_inner(*args, **kwargs):
                 "compile_fx_inner", phase_name="inductor_compile", fwd_only=False
             )
         )
+        # NB: Why is this the dynamo_compile counter?  The rule here is that
+        # if it gets an entry in the dynamo_compile table, we also want to
```

Comment:
Does it make sense to mention "dynamo_compile" here?

Reply:
No, but our table is called dynamo_compile so let's be consistent.